IPython/core/oinspect.py

```diff
@@ -1016,3 +1016,47 @@ def psearch(self,pattern,ns_table,ns_search=[],
             search_result.update(tmp_res)
 
         page.page('\n'.join(sorted(search_result)))
+
+
+def render_signature(obj, oname):
```

Comment:
`obj` is a signature, not the object whose signature we are rendering. Maybe give it a name like `obj_signature`

Reply:
done, thank you!